db/migrate/20230817120303_remove_indexes_from_account_invitations.rb

```diff
@@ -0,0 +1,10 @@
+class RemoveIndexesFromAccountInvitations < ActiveRecord::Migration[6.1]
+  def change
+    remove_index :account_invitations, name: "index_account_invitations_on_account_id"
+    remove_index :account_invitations, name: "index_account_invitations_on_token"
+    remove_index :account_invitations, name: "index_account_invitations_on_user_id"
+
+    remove_foreign_key :account_invitations, column: "account_id"
+    remove_foreign_key :account_invitations, column: "user_id"
```

Comment:
foreign key is another thing - they should be left. please add them in a separate PR (also please make sure they delete cascade)